packages/remediations/src/utils/utils.js

```diff
@@ -116,7 +123,8 @@ export const submitRemediation = (formValues, data, basePath, resolutions) => {
         .then(() => resolver(id, name, false, data.onRemediationCreated));
     } else {
         api.createRemediation({ name: formValues[SELECT_PLAYBOOK], add, auto_reboot: formValues[AUTO_REBOOT] }, basePath)
-        .then(({ id }) => resolver(id, formValues[SELECT_PLAYBOOK], true, data.onRemediationCreated));
+        .then(({ id }) => resolver(id, formValues[SELECT_PLAYBOOK], true, data.onRemediationCreated))
+        .catch(() => resolver(null, formValues[SELECT_PLAYBOOK], true, data.onRemediationCreated, true));
```

Comment:
You are not using the `error` param in resolver. And also maybe add `catch` also to the update remediation case?